component/core/src/main/java/org/exoplatform/social/core/space/SpaceUtils.java

```diff
@@ -1885,23 +1885,28 @@ public static boolean isUserHasMembershipTypesInGroup(String remoteId, String gr
 
 
   /**
-   * Check if the user has the role redactor in that space
+   * Check if the user has the role redactor in that space or that the space is not redactional
    * @param userName
    * @param spaceGroupId
    * @return boolean true if the user has that role Or the role is not present in the space
    * @throws Exception
    */
-  public static boolean isRedactor (String userName, String spaceGroupId) throws Exception{
-    String REDACTOR_MEMBERSHIP_NAME = "redactor";
+  public static boolean isRedactor(String userName, String spaceGroupId) throws Exception {
+    Space space = getSpaceService().getSpaceByGroupId(spaceGroupId);
+    return space != null ? getSpaceService().isRedactor(space, userName) || space.getRedactors().length == 0 : false;
```

Comment:
```suggestion
    return space != null && (getSpaceService().isRedactor(space, userName) || space.getRedactors().length == 0);
```

Reply:
we shouldn't make modification in SpaceUtils because it is used in [https://github.com/exoplatform/agenda/blob/ca1336a87b933634818ac06551ed7ed16d428b2c/agenda-services/src/main/java/org/exoplatform/agenda/util/Utils.java#L412](url), i will check if we haven't spaceHasARedactor in UIcomposer.gtmpl